docs/products/ESPHome-Starter-Kit/learning-the-basics/core-components.md

```
@@ -0,0 +1,70 @@
+- **Including/excluding entities.** Advanced setting that lets you hide specific entities from the web page without removing them from Home Assistant. Useful if you have internal helpers you don't want users to see.
+
+#### Add Component
+
+If you remove the Web Server component, the device still works, but `esphome-starter-kit.local` will stop responding. To add it back:
+
+1. In **ESPHome Device Builder**, open your device and go to **Core configuration**.
+2. Click **Add component**, scroll to **Web Server**, and click **Add**.
+3. Click **Add** once more to confirm.
+4. Toggle **Show advanced settings**, scroll to **Version**, and select **3**.
+
+<a href="https://esphome.io/components/web_server/" target="_blank" rel="noreferrer nofollow noopener" class="md-button md-button--primary"><img src="/assets/esphome-logo.svg" /> ESPHome Web Server docs</a>
```

markdownlint-cli2 (0.22.1)

[warning] Images should have alternate text (alt text)

(MD045, no-alt-text)